cmd/pomerium/main.go

```diff
@@ -213,11 +213,20 @@ func parseOptions(configFile string) (*config.Options, error) {
 	metrics.AddPolicyCountCallback(o.Services, func() int64 {
 		return int64(len(o.Policies))
 	})
-	checksumInt, err := strconv.ParseInt(fmt.Sprintf("0x%s", o.Checksum()), 0, 64)
+
+	// checksumDec, _, err := big.ParseFloat(
+	// 	fmt.Sprintf("0x%s", o.Checksum()),
+	// 	0, 64, big.ToNearestEven)
+	// if err != nil {
+	// 	log.Warn().Err(err).Msg("Could not parse config checksum into decimal")
+	// }
+	// checksumFloat64, _ := checksumDec.Float64()
+	checksumDec, _ := strconv.ParseUint(fmt.Sprintf("0x%s", o.Checksum()), 0, 64)
```

Comment:
errcheck :)

Reply:
`strconv.ParseUint(o.Checksum(), 16, 64)` ?

EDIT: nvm.  TIL base 0 is equivalent if marked with octal prefix
```go
	case base == 0:
		// Look for octal, hex prefix.
		switch {
		case s[0] == '0' && len(s) > 1 && (s[1] == 'x' || s[1] == 'X'):
			if len(s) < 3 {
				return 0, syntaxError(fnParseUint, s0)
			}
			base = 16
			s = s[2:]
		case s[0] == '0':
			base = 8
			s = s[1:]
		default:
			base = 10
		}


```

Reply:
It's actually cleaner to be explicit with the base and avoid the Sprintf().  Changed.

Reply:
👍